OurUmbraco/Documentation/DocumentationUpdater.cs

```diff
@@ -196,6 +196,8 @@ private SiteMapItem GetFolderStructure(DirectoryInfo dir, string rootPath, int l
                             return 3;
                         case "starter-kit":
                             return 4;
+                        case "editor-manual":
```

Comment:
Renamed the folder from `editor-manual` to `editors-manual`

```suggestion
                        case "editors-manual":
```